vector/v.cluster/main.c

```diff
@@ -264,7 +264,7 @@ int main(int argc, char *argv[])
 	    c[2] = 0.0;
 	    n = 0;
 	    sum = sumsq = 0;
-	    min = 1.0 / 0.0;
+		min = NAN;
```

Comment:
This change is not Cmake related and should be separated out

(into a new NAN related PR)